rotkehlchen/api/v1/resources.py

```diff
@@ -3295,3 +3298,31 @@ def patch(self, calendar: CalendarEntry) -> Response:
     @resource_parser.use_kwargs(make_query_schema, location='json_and_query')
     def post(self, filter_query: CalendarFilterQuery) -> Response:
         return self.rest_api.query_calendar(filter_query=filter_query)
+
+
+class CalendarReminderResource(BaseMethodView):
```

Comment:
Same as the endpoint

```suggestion
class CalendarRemindersResource(BaseMethodView):
```